samples/istio/deployments.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: servicea-deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: servicea
+    spec:
+      containers:
+      - name: servicea
+        image: SERVICE_A_TAG:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+        envFrom:
```

Comment:
@Emily-Jiang This is what we were talking about on the call. As it is a `envFrom/configMapRef` it works and exposes the config map as env variables, so all is good.

Reply:
cool. thanks for your confirmation!

Reply:
@pilhuhn please can you approve the other config PR?